src/evm_transition_tool/execution_specs.py

```diff
@@ -102,3 +110,137 @@ def is_fork_supported(self, fork: Fork) -> bool:
         `ethereum-spec-evm` appends newlines to forks in the help string.
         """
         return (fork.transition_tool_name() + "\n") in self.help_string
+
+
+class ExecutionSpecsExceptionMapper(ExceptionMapper):
+    """
+    Translate between EEST exceptions and error strings returned by execution specs t8n.
+    """
+
+    @property
+    def _mapping_data(self):
```

Comment:
Since this is now the default t8n tool to fill tests, I think this list should be complete and correctly populated from this PR. Currently tox is failing because it's not finding the expected exception messages, and I think there are going to be issues when running `fill` like this.

Reply:
the fill works, but the error messages are not provided by exec specs t8n itself. it only prints Failed transaction: 
also if you managed to hit undefined exception, it must be defined yes.

Reply:
But if you run tox at the moment (for the framework with `uvx --with=tox-uv tox -e framework`) it will show a failing test and I think it's related to the exceptions.

Reply:
now it is green